tfhe/src/shortint/engine/server_side/comp_op.rs

```diff
@@ -311,9 +306,8 @@ impl ShortintEngine {
     ) -> EngineResult<()> {
         let modulus = (ct_right.degree.0 + 1) as u64;
         let modulus_msg = ct_left.message_modulus.0 as u64;
```

Comment:
as discussed here I think it makes sense to rename modulus -> modulus_right and modulus_msg -> modulus_left

Reply:
same for other occurrences